Documentation/index.rst

```diff
@@ -62,6 +80,8 @@ get started and experiment with Cilium.
    operations/metrics
    operations/performance/index
    operations/troubleshooting
+   concepts/index
+   installation/microk8s
```

Comment:
microk8s seems a bit random to just be added under operations here.

Reply:
I don't remember the details of that decision, to be honest. I think the idea was to deprecate it because nobody seemed to be using microk8s to run Cilium. @yoyo-go do you remember?

This is probably something we can address as a follow-up anyway.